azerothcore/azerothcore-wotlk
# feat(Core/Formations): Implement GROUP_AI_FLAG_DONT_RESPAWN_LEADER_ON…

…_EVADE flag



## Changes Proposed:
-  Add a flag so the leader doesn't respawn on evade, so it can be used with bosses (see thekal)
-  

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Tested ingame
- 


## How to Test the Changes:


1. Pull thekal
2. kill cat 1, kill thekal, reset
3. notice thekal doesn't respawn, while both cats do
4. kill both cats & reset, notice both cats respawn

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
